src/person/controllers/authenticate_controller.py

```diff
@@ -51,8 +51,9 @@ def create_token_info(self, token_info):
         password = api_settings.AUTH_PASSWORD_SALT + google_user_id
         first_name = token_info.get("given_name")
         last_name = token_info.get("family_name")
+        profile_pic_url = token_info.get("picture")
```

Comment:
`picture` isn't descriptive.. frontend might get confused about whether this is a base 64, another encoding, or a URL

Reply:
"picture" field is from google: https://developers.google.com/identity/sign-in/web/backend-auth

Reply:
wow! til